tests/ci-run-e2e.sh

```diff
@@ -2,14 +2,15 @@
 
 set -xe
 
-if [[ $# -ne 2 ]]; then
-	echo "TEST_CASE TARGET_DRIVER_VERSION are required"
+if [[ $# -ne 3 ]]; then
```

Comment:
We can revert this change once we implement stop invoking this script for upgrading the kernel on a remote instance.